make enquiry.id required (#25)
open-contracting-extensions/ocds_enquiry_extension: README.md

### Unreleased

* Remove the `+partyRole.csv` codelist, whose codes already exist in OCDS 1.1

> * Make `Enquiry.id` a required field for identifier merge

Reviewer: ```suggestion
* Make `Enquiry.id` required so that enquiries are merged by identifier
```
